db/data_migration/20160311093609_generate_content_ids_on_attachments.rb

```diff
@@ -0,0 +1,6 @@
+require 'securerandom'
+
+Attachment.where(content_id: nil).find_each do |attachment|
```

Comment:
The SQL version takes a few seconds as opposed to 20 minutes... It seems that it's an issue with 'statement based replication' which we must be using as opposed to 'row based' which isn't affected. Looks like we're stuck with the 20 minute version then.